tests/test_ml.py

```diff
@@ -130,8 +120,8 @@
 CREATED_UPDATED_MODEL_JSON_1 = {
     'name': MODEL_NAME_1,
     'displayName': DISPLAY_NAME_1,
-    'createTime': CREATE_TIME_JSON,
-    'updateTime': UPDATE_TIME_JSON,
+    'createTime': CREATE_TIME,
```

Comment:
Can these 2 values ever be not set in response? At least the parsing logic assumes so. Might want to add a test case for it.

Reply:
They will always be set in the response.